libs/langchain/langchain/chains/graph_qa/neptune_cypher.py

```diff
@@ -29,6 +58,17 @@ def extract_cypher(text: str) -> str:
     return matches[0] if matches else text
 
 
+def use_simple_prompt(llm: BaseLanguageModel) -> bool:
+    """Decides whether to use the simple prompt"""
+    return llm._llm_type and "anthropic" in llm._llm_type  # type: ignore
```

Comment:
can probably use `isinstance` to be more direct?

Reply:
@hwchase17 
For `isinstance` check, we will need imports for all classes `AnthropicChat`, `Anthropic`, `Bedrock` and `BedrockChat`. I want to avoid imports to any specific LLMs here, but open to your suggestion.